docs/dashboard.asciidoc

```diff
@@ -1,2 +1,117 @@
+[float]
 [[dashboard]]
-== Working with Dashboards
\ No newline at end of file
+== Dashboard
+
+A Kibana _dashboard_ displays saved visualizations in groups that you can arrange. You can save a dashboard to share or 
+reload at a later time.
+
+[float]
+[[getting-started]]
+=== Getting Started
+
+You need at least one saved <<visualize, visualization>> to use a dashboard.
+
+[float]
+[[creating-a-new-dashboard]]
+==== Creating a New Dashboard
+
+The first time you click the *Dashboard* tab, Kibana displays an empty dashboard.
+
+image:images/NewDashboard.png[New Dashboard screen]
+
+Add visualizations to start creating your dashboard.
+
+[float]
+[[adding-visualizations-to-a-dashboard]]
+==== Adding Visualizations to a Dashboard
+
+To add a visualization to the dashboard, click the *Add Visualization* image:images/AddVis.png[Plus] button in the 
+toolbar panel. Select a saved visualization from the list. You can filter the list of visualizations by typing a filter 
+string into the *Visualization Filter* field.
+
+The visualization you select appears in a _container_ on your dashboard.
+
+*NOTE:* If you see a message about the container's height or width being too small, <<resizing-containers,resize the 
+container>>.
+
+[float]
+[[saving-dashboards]]
+==== Saving Dashboards
+
+To save the dashboard, click the *Save Dashboard* button in the toolbar panel, enter a name for the dashboard in the *Save As* field, and click the *Save* button.
+
+[float]
+[[loading-a-saved-dashboard]]
+==== Loading a Saved Dashboard
+
+Click the *Load Saved Dashboard* button to display a list of existing dashboards. The saved dashboard selector includes a text field to filter by dashboard name and a link to the Object Editor, accessible through *Settings > Edit Saved 
+Objects*, to manage your saved dashboards.
```

Comment:
I'd split this into two sentences--one that says what the object editor is, and one that says you can also directly access the object editor through Settings > Objects. (BTW, my inclination is to reference the pages by the tab labels, rather than the page titles.)